omero/sysadmins/config.rst

```diff
@@ -1978,7 +2017,23 @@ omero.web.logdir
 ^^^^^^^^^^^^^^^^
 A path to the custom log directory.
 
-Default: `/home/omero/OMERO.server/var/log`
+Default: `/Users/jmarie/Documents/git-repo/openmicroscopy/dist/var/log`
```

Comment:
This has `jmarie` now.

Reply:
let me go back in history. Those files should now be there